easybuild/easyblocks/c/cuda.py

```diff
@@ -124,6 +133,11 @@ def create_wrapper(wrapper_name, wrapper_comp):
 
     def sanity_check_step(self):
         """Custom sanity check for CUDA."""
+
+        versionfile = read_file(os.path.join(self.installdir, "version.txt"))
```

Comment:
@wpoely86 At the very least, this should be conditional based on the CUDA version. This `version.txt` file is not there for CUDA 7.0.28 for example, so this change is not backward-compatible...

Also, it really only makes sense to check when patches are involved, i.e. when `len(self.src) > 1`?

Reply:
fixed in #1483